test/setup.sh

```diff
@@ -17,7 +17,7 @@ go get \
   github.com/mattn/goveralls \
   github.com/modocache/gover \
   github.com/tools/godep \
-  golang.org/x/tools/cover &
+  golang.org/x/tools/cover
 
 # Create the database and roles
 ./test/create_db.sh &
```

Comment:
Since the `go get` commands are going to take the longest to run leave the `&` on L20 and just move `./test/create_db.sh` below the `wait` on L33.